source/common/http/http1/codec_impl.cc

```diff
@@ -462,12 +497,17 @@ void ConnectionImpl::onHeaderField(const char* data, size_t length) {
 }
 
 void ConnectionImpl::onHeaderValue(const char* data, size_t length) {
-  if (header_parsing_state_ == HeaderParsingState::Done) {
+  const absl::string_view header_value = absl::string_view(data, length);
+  ENVOY_CONN_LOG(trace, "onHeaderValue: data={}", connection_, header_value);
+
+  if (header_parsing_state_ == HeaderParsingState::Done && !enable_trailers_) {
```

Comment:
can onHeaderValue happen before onHeaderField?  Basically can this be called with state done?

Reply:
According to the table at the end of this section https://github.com/nodejs/http-parser/blob/master/README.md#callbacks it _should_ not happen.

Reply:
Maybe make it an ASSERT then, and trust the fuzzers?

Reply:
> Basically can this be called with state done?

Actually sorry, this can happen. It will happen since the parser will continue to call the callbacks even when the enable_trailers is false.